src/patternfly/components/NotificationBadge/docs/code.md

```diff
@@ -0,0 +1,17 @@
+## Overview
+
+Always add a modifier class. Never use the class `.pf-c-notification-badge` on its own.
+
+## Accessibility
+
+Be sure that the component associated with this indicator handles screenreader text indicating read or unread notifications.
+
+
+## Usage
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-notification-badge` | `<div>` |  Initiates a notification badge. **Always use it with a modifier class.** |
```

Comment:
I wonder if we should suggest this be a `<span>` instead? Seems like it would almost always be used inline to wrap an icon.

Reply:
changed.